integration-tests/pytests/test_client_cli.py

```diff
@@ -223,7 +223,8 @@ def test_deposit_to_other_wallet():
 
 @pytest.mark.zerofee
 def test_withdraw_to_other_wallet():
-    time.sleep(3)
+    # the tx-query cert_validation is 200 (setted in .drone.yaml), check the cert refresh or not
+    time.sleep(210)
```

Comment:
how is this checked / asserted? If it's "implicitly" tested that one can connect, it can still be the case that the cert has a long validity, so nothing was really tested -- it's preferable to have a dedicated test where one would dump and check the certificate (`openssl s_client -connect <tx-query-host:port> -state -debug`  equivalent in python)